middleware/LoggingHandler.md

```diff
@@ -1,11 +1,22 @@
-# Logging Handler [In Progress]
+# Logging Handler [Draft 2]
 
 ## Objectives
+<ul>
+ <li>This feature is a middleware component that allows developers to plug in their own 3rd party logging platform to their application, currently it enables developers to log items in the graph SDK related to: request to graph service latency, authentication latency, redirect count, exceptions, request and client id’s. These items will be provided to a third-party logging framework through various exporters. List of items is open-ended and can be changed.</li>
+</ul>
 
-This component should allow a developer to provide their own logging listener and important information related to the request and response should be written to the listener.  Where the native HTTP client library has its own built in logging infrastructure, that should be preferred over a custom solution.
-
-## Requirements
-
+## Requirements 
+ <ul>
```

Comment:
Will the logging handler always be present? 
How will the LoggingHandler be created? 
Do we pass an ILogger and/or OpenCensus Tracer to the logging handler?
What dependencies will the LoggingHandler have?
What requirements will we put on our customers to make use logging handler?
Can you pseudo-code how a customer might instantiate or configure a LoggingHandler?
How do we configure the logging handler?